nanoDFT/experimental_pmap_nanoDFT.py

```diff
@@ -2,14 +2,8 @@
 import jax
 import jax.numpy as jnp
 import numpy as np
-import os
-os.environ['OMP_NUM_THREADS'] = "16"
 import pyscf
 from jsonargparse import CLI, Namespace
-import sys
-sys.path.append("../")
-import os 
-os.environ['TF_POPLAR_FLAGS'] = """--executable_cache_path=/tmp/pyscf-ipu-cache/"""
```

Comment:
I'm on the fence as to whether we want to cache by default. One solution: add a `--cache` option and discuss whether it should be true/false by default.

Reply:
See https://github.com/graphcore-research/pyscf-ipu/pull/30